src/sentry/api/bases/sentryapps.py

```diff
@@ -113,9 +113,11 @@ class SentryAppPermission(SentryPermission):
     unpublished_scope_map = {
         'GET': ('org:read', 'org:integrations', 'org:write', 'org:admin'),
         'PUT': ('org:read', 'org:integrations', 'org:write', 'org:admin'),
+        'POST': ('org:write', 'org:admin'),  # used for publishing an app
         'DELETE': ('org:write', 'org:admin'),
     }
 
+    # no POST route for a published app yet
```

Comment:
We should include the permissions in here anyway. Someone may request this endpoint (outside of the app) with a published integration.